### PR TITLE
Enum to bool converter

### PR DESCRIPTION
**Pull request by VladislavAntonyuk**
https://github.com/CommunityToolkit/Maui/issues/58

### CODE REVIEW

**Comment by PRMerger6** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Index File | This PR contains a change to the index file.

**Comment by PRMerger3** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Index File | This PR contains a change to the index file.

**Comment by PRMerger20** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Index File | This PR contains a change to the index file.

File: docs/maui/converters/enum-to-bool-converter.md [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+			Label.IsVisibleProperty,
+			new Binding(
+				nameof(ViewModel.SelectedPlatform),
+				converter: new EnumToBoolConverter(),
+                converterParameter: MyDevicePlatform.Tizen));
+
+		Content = new VerticalStackLayout
+        {
+            Children = { picker, label }
+        };
```
Comment (bijington) [inline docs/maui/converters/enum-to-bool-converter.md:122]: @jfversluis are you able to check if this renders correctly in the preview please?

**Comment by PRMerger20** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Index File | This PR contains a change to the index file.

**Review by bijington — APPROVED** [review]
Thanks for this @VladislavAntonyuk. I just added the C# samples and made some minor tweaks to some text.

File: docs/maui/converters/enum-to-bool-converter.md (hunk 1 above)
Comment (jfversluis) [reply docs/maui/converters/enum-to-bool-converter.md:122]: Looks good to me!

**Pull request merged**